src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__imac-photo {
+    display: block;
+    width: 100%;
+    height: 134px;
+    margin-top: 32px;
+    padding: 0 3px;
+  }
+
+  &__name {
+    width: #{$weight-card};
+    margin-top: 40px;
+    align-items: center;
+
+    font-size: inherit;
+    line-height: 18px;
+    font-weight: 500;
+    color: #{$color-darkblue};
+  }
+
+  &__code {
+    margin-top: 4px;
+    align-items: center;
+
+    font-size: 10px;
+    font-weight: 400;
+    color: #616070;
+  }
+
+  &__review {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    align-content: center;
+
+    width: #{$weight-card};
```

Comment:
You can write this variable without #{}, it's redundant. Please delete it in other places.